### PR TITLE
docs: change prismjs theme

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: docs/website/src/components/shared/Content.js [hunk 1]
```diff
@@ -99,6 +90,17 @@ const useStyles = theme => ({
         color: theme.link.normal,
       },
     },
+    '& .gatsby-highlight pre': {
+      background: theme.code.main,
+      '& .token.operator, .token.entity, .token.url, .language-css .token.string, .style .token.string': {
```
Comment (wdavidw) [inline docs/website/src/components/shared/Content.js:95]: Please leave a comment explaining why

File: docs/website/src/pages/index.js [hunk 2]
```diff
@@ -89,6 +90,16 @@ const useStyles = theme => ({
   feature: {
     paddingBottom: '0 !important',
   },
+  code: {
+    background: '#f0f0f5',
+    padding: '1em',
+    margin: '.5em 0',
+    overflow: 'auto',
+    '& .token.operator, .token.entity, .token.url, .language-css .token.string, .style .token.string': {
```
Comment (wdavidw) [inline docs/website/src/pages/index.js:98]: same, add a comment

**Review by wdavidw — CHANGES_REQUESTED** [review]
Please add comments

**Comment by sergkudinov** [comment]
> Please add comments

done

**Pull request merged**